.github/workflows/packaging.yml

```diff
@@ -205,6 +205,22 @@ jobs:
         with:
           fetch-depth: 0 # We need full history for versioning
           submodules: recursive
+      - name: Set Sentry telemetry env vars
+        id: set-telemetry-env-vars
+        run: |
+          if [[ "${{ github.repository }}" == 'netdata/netdata' && \
+                  "${{ matrix.bundle_sentry }}" == 'true' && \
+                  "${{ github.event_name }}" == 'workflow_dispatch' ]]; then
+
+              echo "RELEASE_PIPELINE=Production" >> $GITHUB_ENV
+              echo "UPLOAD_SENTRY=true" >> $GITHUB_ENV
+
+          else
+
+              echo "RELEASE_PIPELINE=Unknown" >> $GITHUB_ENV
+              echo "UPLOAD_SENTRY=false" >> $GITHUB_ENV
+
+          fi
```

Comment:
This is still not resolved in the current version of the PR. At minimum, the actionlint warnings are a hard blocker for merging, because they are trivial to fix.

Reply:
thanks Austin